### PR TITLE
Fix some issues

### PR DESCRIPTION
**Pull request by appinteractive**
- Fix Server Side Rendering issue with media-query 
- Fixed MenuItem slot in Menu component, now you can overwrite it properly
- Use kebab case for prop name documentation to adhare to vue/html attribute standard
- Bigger search, easier use
- Accessbility for inputs by adding name prop when specified
- Disable autocomplete on select to prevent weired usabillity issues
- Better disabled state for inputs so they are recognized better

### CODE REVIEW

File: src/system/components/data-input/shared/input.scss [hunk 1]
```diff
@@ -37,6 +37,7 @@
       opacity: $opacity-disabled;
       pointer-events: none;
       cursor: not-allowed;
+      background-color: $color-neutral-80;
```
Comment (visualjerk) [inline src/system/components/data-input/shared/input.scss]: Please add a new background-color token in `tokens/color.yml` named background-color-disabled and use it here. We should keep the separation between the whole color palette (color choices) and colors for specific use cases (color decisions).

File: src/system/components/typography/Chip/style.scss [hunk 2]
```diff
@@ -47,15 +48,19 @@
 
 .ds-chip-size-base {
   font-size: $font-size-small;
+  padding-left: $space-small * 0.9;
+  padding-right: $space-small * 0.9;
```
Comment (visualjerk) [inline src/system/components/typography/Chip/style.scss]: The whole purpose of design tokens gets negated by using custom computations. Please just use $space-small instead.

File: src/system/components/typography/Chip/style.scss [hunk 3]
```diff
@@ -47,15 +48,19 @@
...
 }
 
 .ds-chip-size-small {
   font-size: $font-size-x-small;
-  padding: $space-xxx-small $space-x-small;
+  padding: $space-xxx-small $space-x-small * 1.2;
```
Comment (visualjerk) [inline src/system/components/typography/Chip/style.scss]: Same as above. Please just use $space-x-small or $space-small instead.

File: src/system/mixins/media-query.js [hunk 4]
```diff
@@ -23,10 +23,15 @@ function initListener() {
...
-    updateWindowSize()
+  try {
+    if (window && typeof window !== 'undefined') {
+      window.addEventListener('resize', updateWindowSize)
+      updateWindowSize()
+    }
+    init = true
+  } catch (err) {
     init = true
+    return false
```
Comment (visualjerk) [inline src/system/mixins/media-query.js]: It looks like it breaks the media query function all together, when loaded with SSR. Wouldn't it make more sense to add checks for everything necessary for updateWindowSize to work?

File: src/system/components/typography/Chip/style.scss (hunk 2 above)
Comment (appinteractive) [reply src/system/components/typography/Chip/style.scss]: Yeah I’m also unhappy with it but I’d does not look right. I just miss some pixels to get it right. 🤔

File: src/system/components/typography/Chip/style.scss (hunk 3 above)
Comment (appinteractive) [reply src/system/components/typography/Chip/style.scss]: Nether looks right. 😕

File: src/system/mixins/media-query.js (hunk 4 above)
Comment (appinteractive) [reply src/system/mixins/media-query.js]: Im open for a better solution as that’s like a hack somehow. I was thinking if the media query should be used on the grid at all but just on the components. That would fix the jumping layout on ssr load I think. Any suggestions?

File: src/system/components/typography/Chip/style.scss (hunk 2 above)
Comment (appinteractive) [reply src/system/components/typography/Chip/style.scss]: Maybe I need to work here with em values instead.

Comment (visualjerk) [reply src/system/components/typography/Chip/style.scss]: That would be a viable option. Please have a look at the font-spacing tokens then.

**Pull request merged**